Wox.Core/Configuration/Portable.cs

```diff
@@ -0,0 +1,200 @@
+using Microsoft.Win32;
+using Squirrel;
+using System;
+using System.IO;
+using System.Reflection;
+using System.Windows;
+using Wox.Infrastructure;
+using Wox.Infrastructure.Logger;
+using Wox.Infrastructure.UserSettings;
+using Wox.Plugin.SharedCommands;
+
+namespace Wox.Core.Configuration
+{
+    public class Portable : IPortable
+    {
+        private UpdateManager portabilityUpdater;
+
+        public void DisablePortableMode()
+        {
+            portabilityUpdater = new UpdateManager(string.Empty, Constant.Wox, Constant.RootDirectory);
```

Comment:
Can we maybe move this to the constructor? or at least only create it in one place?

Reply:
So the reason I put it to local is following the code in updater how they are using Squirrel, and following this [issue](https://github.com/Squirrel/Squirrel.Windows/pull/1396). 

The update manager is newed up during app start up so local helps manage it better in terms of using it when required and disposing it straight away as oppose to newing it up in constructor and keeping it along the life the app even if it never gets used.

Reply:
Updated now, see commit